### PR TITLE
Submodule updateV3

### PR DESCRIPTION
**Pull request by mokaddem**
Improve overall feedback - WiP

### CODE REVIEW

**Comment by iglocska** [comment]
It looks good to me!

**Comment by mokaddem** [comment]
> It looks good to me!

Was adding some finishing touches :)
There you go.

**Comment by mokaddem** [comment]
Let me know if something seems counter-intuitive. :)

File: app/Model/Server.php [hunk 1]
```diff
@@ -4624,25 +4626,30 @@ public function getSubmoduleGitStatus($submodule_name, $superproject_submodule_c
...
         return $status;
     }
 
     public function updateSubmodule($submodule_name=false) {
         $path = APP . '../';
         if ($submodule_name == false) {
-            $command = sprintf('cd %s; git submodule update', $path);
-            exec($command, $output);
+            $command = sprintf('cd %s; git submodule update 2>&1', $path);
+            exec($command, $output, $return_code);
```
Comment (SteveClement) [inline app/Model/Server.php:4643]: Beautiful ;)

**Comment by mokaddem** [comment]
I still need to add the db update after the pull.
But as it is confusing for users, i'm opening the PR.

**Comment by SteveClement** [comment]
Seems to work in my tests.

**Pull request merged**